versioned_docs/version-2.x/advanced/analytics/server-side-tagging-tracking.mdx

```diff
@@ -0,0 +1,186 @@
+---
+sidebar_position: 4
+title: Server-side tracking / tagging
+description:
+  This page will guide you trough the setup of Server-side tracking and tagging
+  services
+---
+
+<p>{frontMatter.description}</p>
+
+## Addingwell
+
+### Requirements
+
+To use Addingwell, you must have create an
+[Addingwell account](https://www.addingwell.com/)
+
+### Setup Addingwell server
+
+You must follow the official
+[Addingwell Get Started](https://docs.addingwell.com/b/2CA8BCF1-E67D-46DF-A2BE-F93017D8FD7A/Getting-started-with-Addingwell)
+documentation.
+
+### With GTM
+
+This guide will show you how to integrate Addingwell with Google Tag Manager.
+This way, the Analytics tag will be implemented with Google Tag Manager but the
+traffic will head to your custom server
+
+#### Setup GTM
+
+You must follow the
+[Addingwell Send data documentation (Option 1)](https://docs.addingwell.com/b/C45DD221-2773-4DE0-B588-39B60785291A/Send-the-data-to-your-tagging-server)
+
+#### Setup GTM Analytics plugin in Front-Commerce
```

Comment:
**nitpick:** since it is similar to what is already documented, I'd link to the existing section to prevent duplication

**suggestion:** reword into something more concise, such as

```suggestion
#### Configure Google Tag Manager in Front-Commerce

Follow the [Google Tag Manager](/docs/2.x/advanced/analytics/plugins#google-tag-manager) installation procedure documented above.

To use it with Addingwell, you'll have one more step to do: configure …………
```